### ISSUE TITLE
Only showing 40/437 followers

### ISSUE THREAD

**Issue by seanlynch** [magicalraccoon/tootstream#210]
I have 437 followers, but for some reason the "followers" command in tootstream is only showing 40. The "info" command shows the correct number of followers. Version is 0.3.6.r7.gf2f8b8d-1, built from the "tootstream-git" AUR package on Arch Linux.

**Comment by seanlynch** [comment]
Trying with `madonctl account followers` with no --max-id or --since-id also gives me exactly 40 results, so this is almost certainly an API limit. I'd be fine with some kind of pagination since I'm not sure what I"ll do with an unpaginated list of 437 accounts anyway.

**Comment by craigmaloney** [comment]
Right. That was my initial guess as to what was going on, but I'm not sure how best to paginate through all of these from the tootstream interface.  This will also likely apply to other paginated things like notifications as well.
